config/environments/development.rb

```diff
@@ -36,6 +36,18 @@
   # Raises helpful error messages.
   config.assets.raise_runtime_errors = true
 
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_host_name: 's3-us-west-2.amazonaws.com',
+    s3_credentials: {
+      bucket: ENV['AWS_BUCKET_NAME'],
+      access_key_id: ENV['AWS_ACCESS_KEY_ID'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.